src/main/java/anichan/commands/BookmarkCommand.java

```diff
@@ -38,23 +52,19 @@
     private static final String BOOKMARK_EXECUTE_ADD_NOTE = "Executing bookmark add note.";
     private static final String BOOKMARK_EXECUTE_REMOVE_NOTE = "Executing bookmark remove note.";
     private static final String BOOKMARK_EXECUTE_SUCCESS = "Execute Bookmark command successful.";
-    private static final String BOOKMARK_LIST_HEADER = "Listing all anime in bookmark:";
-    private static final String BOOKMARK_INFO_HEADER = "Here is the information for that anime.";
+
+    //Constant Parameter
+    private static final int ZERO_PARAM = 0;
     private static final String BOOKMARK_NOTE_FORBIDDEN_CHAR = "~";
```

Comment:
Would it be better to use regex here?

Reply:
I think its only checking for a single character, it would not make much difference. Thanks.